13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+
+While this book is focused on applying data science to education, an important consideration for data science is how to teach others about it. This is particularly the case for a book that is by a team of authors who are involved in education. Also, we expect readers of this book - many who will also be involved in education - will be interested in teaching others about data science.
+
+This chapter, then, focuses on teaching data science to others. This chapter is organized around three topics, which progress from concrete and specific (with reference to pedagogical approach we used to guide how we wrote this book) to more general ideas and findings from educational research about teaching and learning.
+
+<!-- Not sure this organization works - may want to move general strategies first or second, and have teaching data science be last? -->
+
+1. The pedagogical principles this book is based upon
+1. Strategies for teaching data science
+1. General strategies related to teaching and learning
+
+## The pedagogical principles this book is based upon
+
+As the authors of a book about data science in education - and readers of books that taught us about data science - we considered what would make it effective for our readers when we set out to write it. The result of this process was a pedagogical framework that consists of three principles: problem-based learning, differentiation, and universal design. We consider each of these in turn.
+
+### Problem-based learning 
+
+Problem-based learning (PBL) is a method of instruction that presents learners with a real-world challenge in which they must apply their skills and knowledge to solve. We applied this principle to the design of this book through including walk-throughs for common questions with respect to data science and education. Whereas some topics may benefit less from such an approach (and the inclusion of walk-throughs), for data science, we believe this is important because we do not have all of the right answers in this text. Moreover, there is not one right statistical model or algorithm, a technique to write code or even software and tools to utilize. 
+
+Thus, the text features walkthroughs that reflect the types of challenges that educational data scientists may encounter in the course of their work: readers may choose to go about approaching the analysis of the data used in each walkthrough differently. At the end of walkthrough, there exist exercises that provide the opportunity for readers to extend and apply the ideas presented in the chapter. Moreover, the challenges are structured in such a way that readers return to some of them, but with different aims, over the course of the book.
+
+<!-- To what extent do we need to change the walkthroughs or their order to make this accurate? -->
```

Comment:
The walk-throughs - at this point in time - all have what I would consider "good core content." That is, they have the central aim of the walk-through written down, but will benefit from editing both in terms of overall arcs of the book as well as adding content that will more accurately reflect this statement.

Reply:
adding this as a note to doc to (consistently) return to